pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1592,3 +1600,10 @@ func (c *VolumeBindingChecker) predicate(pod *v1.Pod, meta algorithm.PredicateMe
 	glog.V(5).Infof("All PVCs found matches for pod %v/%v, node %q", pod.Namespace, pod.Name, node.Name)
 	return true, nil, nil
 }
+
+// RegisterPodFitsResourcesPredicateMetadataProducer registers a PredicateMetadataProducer used by PodFitsResources
+func RegisterPodFitsResourcesPredicateMetadataProducer(scalaResourcesHandleByExtenders sets.String) {
+	RegisterPredicateMetadataProducer("PodFitsResources", func(pm *predicateMetadata) {
```

Comment:
Please use the `const` name of the predicate instead of the string literal ("PodFitsResources").